dart/dynamics/Entity.h

```diff
@@ -63,11 +63,23 @@ class Shape;
 /// may have different policies about how/if their reference frame or name
 /// can be changed. Use the Detachable class to create an Entity whose reference
 /// Frame can be changed arbitrarily.
-class Entity
+class Entity : public common::Subscription
 {
 public:
   friend class Frame;
 
+  enum Notification {
+
+    FRAME_CHANGE_NOTICE = 1,
```

Comment:
I prefer to use tense to represent a name of event (notification), for example, `FRAME_CHANGING` and `FRAME_CHANGED`. `FRAME_CHANGING` tells the subscribers that the frame will be changed and `FRAME_CHANGED` it's already changed.

```cpp
sendNotification(FRAME_CHANGING);

// Change the frame

sendNotification(FRAME_CHANGED);
```

For now, `FRAME_CHANGING` is not necessary but I'd like to use the name `FRAME_CHANGED` instead of `FRAME_CHANGE_NOTICE`.